clases/rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly include `person` and `book` in the setter method so a user can make rentals with each param

Reply:
Well solved